imap/cyr_expire.c

```diff
@@ -326,9 +380,16 @@ static int archive(const mbentry_t *mbentry, void *rock)
     if (mailbox_open_iwl(mbentry->name, &mailbox))
         goto done;
 
-    verbosep("archiving mailbox %s\n", mbentry->name);
+    if (!arock->skip_annotate &&
+        get_annotation_value(mbentry->name, IMAP_ANNOT_NS "archive",
```

Comment:
... but then this usage of `get_annotation_value()` seems to expect non-zero on success, the same as `parse_duration()`